app/components/embed/media_with_companion_windows_component.html.erb

```diff
@@ -161,11 +161,13 @@
 
   <dialog class="modal-container" data-media-target="accessibility" data-action="click->media#handleBackdropClicks" id="accessibility">
     <header>Accessibility</header>
-    <p>To request a transcript or other accommodation for this media item,
-      please send a request along with a link to this item to
-      <a href="mailto:purl-feedback@lists.stanford.edu">purl-feedback@lists.stanford.edu</a>.</p>
-
-    <p>A transcript may be available for download. Click the share & download icon to see if a download is available.</p>
+    <% if transcript_files.present? %>
```

Comment:
Do we need to make sure the transcript is downloadable by the user?

Reply:
I mean to say, the transcript could be "stanford only" but the user may not have that access.  Might need design for that.

Reply:
It has in `models/embed/purl.rb` - which I believe is confirmed in the test, but if you think more is necessary, I'm open to suggestion.

Reply:
I would need some pairing to think through that.